Model/Transaction.php

```diff
@@ -373,6 +373,19 @@ protected function getParentAmounts($attr, $items, $type = 'order') {
      */
     protected function getProductTaxCode($item, $order)
     {
+        // Check for a PTC saved to the Item
+        // For configurable products, load the PTC of the child
+        if ($item->getHasChildren() && $item->getProductType() == 'configurable') {
+            $child = $item->getChildrenItems()[0];
```

Comment:
@ryanreeves-taxjar Do we need to use `reset` here similar to #165?

Reply:
I think that's a good idea.